env0/resource_environment.go

```diff
@@ -524,16 +532,20 @@ func resourceEnvironmentCreate(ctx context.Context, d *schema.ResourceData, meta
 		// setEnvironmentSchema() (several lines below) sets the blueprint id in the resource (under "without_template_settings.0.id").
 		environment, err = apiClient.EnvironmentCreateWithoutTemplate(payload)
 	}
+
 	if err != nil {
 		return diag.Errorf("could not create environment: %v", err)
 	}
+
 	environmentConfigurationVariables := client.ConfigurationChanges{}
-	if environmentPayload.DeployRequest.ConfigurationChanges != nil {
-		environmentConfigurationVariables = *environmentPayload.DeployRequest.ConfigurationChanges
+	if environmentPayload.ConfigurationChanges != nil {
```

Comment:
The way it was done before isn't correct.
There may be a use-case for `*environmentPayload.DeployRequest.ConfigurationChanges`, but I couldn't find it while playing with the UI.

Reply:
This is for "create" use-case.
The "redploy change" I made a bit below would now make this always nil.

Looking at how it works in the UI: the deploy request does not contain the configuration. Just a few fields related to the blueprint/template. This makes sense because all the configurations are already set when creating the environment.

Reply:
looked at the payload, got your point, the variables are set in their own field, not in the field `deployRequest`. thanks